module1/projects/connect_four/iteration_1.md

```diff
@@ -10,30 +10,116 @@ _[Back to Requirements](./requirements)_
 
 In this iteration, you are required to use TDD to create your classes. Use the interaction pattern to determine what a method should do and write one or more tests to verify that expected behavior. Then you can implement the method. You should always write code with the purpose of making a test pass.
 
-## Printing the Board
+## Cell 
 
-When a user runs the command to start the game, they will see a welcome message, followed by an empty board. The board itself will represent empty spaces with periods and column names with a letter A-G.
+A Cell object represents the space where a chip could be placed on the Connect 4 board. We will be able to read its column letter and row number, as well as its display value. It will also have behavior to fill in the cell with a new value.
 
-```
-ABCDEFG
-.......
-.......
-.......
-.......
-.......
-.......
+The Cell class should follow this interaction pattern:
+
+```ruby
+pry(main)> require './lib/cell'
+# => true
+pry(main)> a1 = Cell.new("A1")
+# => #<Cell:0x000000014a232490 @column="A", @filled=false, @placement="A1", @row=1, @value=".">
+pry(main)> b3 = Cell.new("B3")
+# => #<Cell:0x000000014a29b080 @column="B", @filled=false, @placement="B3", @row=3, @value=".">
+pry(main)> a1.placement
+# => "A1"
+pry(main)> a1.column
+# => "A"
+pry(main)> a1.row
+# => 1
+pry(main)> a1.value
+# => "."
+pry(main)> a1.filled?
+# => false
+pry(main)> a1.fill("X")
```

Comment:
When calling `Cell#fill` are we expecting it to return the value or just perform the fill and we can get the value with a call to `Cell#value`? If it is the latter then the interaction pattern would need to be updated.